org-tanglesync.el

```diff
@@ -302,13 +302,19 @@ Only takes effect when :custom is set"
   (interactive)
   (org-tanglesync-process-buffer t))
 
+(defun tangled-p ()
```

Comment:
so this function already exists at #L105, where it tries to retrieve the `tfile` from the `:tangle`  keyword and returns nil if it cannot be found